client/js/new-user.js

```diff
@@ -0,0 +1,56 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const form = document.querySelector("#formy-mcformface")
+    form.addEventListener("submit", userCheck)
+})
+
+const userNotFound = async () => {
+    const firstname = document.querySelector("#firstname").value 
+    console.log(firstname)
+    const lastname = document.querySelector("#lastname").value
+    console.log(lastname)
+    const age = document.querySelector("#age").value
+    console.log(age)
+    let response = await axios.post(`http://localhost:11000/users/`, {firstname, lastname, age});
+    enterSite()
+}
+
+const userFound = () => {
+    let form = document.querySelector("#formy-mcformface")
+    let alertUser = document.createElement("p")
+    alertUser.innerText = "User Already Exists!"
+    let linkToLogin = document.createElement("a")
+    linkToLogin.innerText = "Click Here To Login!"
+    linkToLogin.href = "../html/login.html"
+    form.appendChild(alertUser)
+    form.appendChild(linkToLogin)
+}
```

Comment:
in the future, when you're appending more than one child to the same element, you can do
form.append(alertUser, linkToLogin) instead of two appendChilds